### ISSUE TITLE
Detect pre-formatted log messages

### ISSUE THREAD

**Issue by adamchainz** [adamchainz/flake8-logging#19]
### Description

Logging calls are designed to do potentially-costly string formatting lazily, to avoid wasting time when the message will be ignored due to the current level or filters. Log messages that pre-format messages are counter to this:

```python
logging.info(f"Took {elapsed:.2f} seconds")
```

Let’s detect log message calls with a message constructed through f-string, `str.format()` call, or `str % params`, and no arguments, and recommend using logging formatting instead.

Basically, duplicate the titular feature of [flake8-logging-format](https://pypi.org/project/flake8-logging-format/), but use a single error code for all formatting options.

**Comment by adamchainz** [comment]
Another thing to mention: pre-formatted log messages are harder to group in error-capture tools, such as Sentry. I think Sentry has heuristics to try and group preformatted messages but it's not perfect.
